turbopack/crates/turbo-tasks-backend/src/backend/mod.rs

```diff
@@ -0,0 +1,1084 @@
+pub mod indexed;
+mod operation;
+mod storage;
+
+use std::{
+    borrow::Cow,
+    collections::{HashMap, HashSet},
+    future::Future,
+    hash::BuildHasherDefault,
+    mem::take,
+    pin::Pin,
+    sync::{
+        atomic::{AtomicUsize, Ordering},
+        Arc,
+    },
+    time::Duration,
+};
+
+use anyhow::{bail, Result};
+use auto_hash_map::{AutoMap, AutoSet};
+use dashmap::DashMap;
+pub use operation::AnyOperation;
+use operation::{
+    get_aggregation_number, is_root_node, AggregatedDataUpdate, AggregationUpdateJob,
+    AggregationUpdateQueue, CleanupOldEdgesOperation, ConnectChildOperation, OutdatedEdge,
+};
+use parking_lot::{Condvar, Mutex};
+use rustc_hash::FxHasher;
+use smallvec::smallvec;
+use turbo_tasks::{
+    backend::{
+        Backend, BackendJobId, CachedTaskType, CellContent, TaskExecutionSpec, TransientTaskRoot,
+        TransientTaskType, TypedCellContent,
+    },
+    event::EventListener,
+    registry,
+    util::IdFactoryWithReuse,
+    CellId, FunctionId, RawVc, ReadConsistency, TaskId, TraitTypeId, TurboTasksBackendApi,
+    ValueTypeId, TRANSIENT_TASK_BIT,
+};
+
+use self::{operation::ExecuteContext, storage::Storage};
```

Comment:
Makes sense, but in this case we should make it a lint rule, or cargo fmt config, or something else autofixable

Reply:
I've looked through the lints that clippy has, and the options that rustfmt has, and wasn't able to find anything. 😞

This might be tricky for them to do because to do this correctly, this may require some cross-module information to understand where this file is in the module tree? Not sure.